core/src/SYCL/Kokkos_SYCL_Instance.cpp

```diff
@@ -126,6 +126,11 @@ void SYCLInternal::initialize(const sycl::queue& q) {
 
     m_maxThreadsPerSM =
         d.template get_info<sycl::info::device::max_work_group_size>();
+    // FIXME_SYCL this should give the correct value for NVIDIA GPUs
+    m_maxConcurrency =
+        m_maxThreadsPerSM * 2 *
```

Comment:
Why 2x?

Reply:
is max_compute_units correct? Or does that provide the number of CUs (which would only be 80, instead of 80*2048).

Reply:
The concurrency computed here is 1024*2*80 for NVIDIA GPUs.

Reply:
> Why 2x?

m_maxThreadsPerSM is currently the maximum workgroup size which happens to be 1024 for Volta70. The maximum number of threads per streaming multiprocessor is 2048, though.
We should probably just rename the variable for SYCL to avoid confusion.